### ISSUE TITLE
Configuration of new inbound email account imports all emails

### ISSUE THREAD

**Issue by psatasiya** [salesagility/SuiteCRM#1715]
Hello,
When I'm setting up new inbound email account in any version of suiteCRM,it imports all the emails from that account to our CRM.I know it's suiteCRM's default behavior,
but it would be good if it will ask for certain criteria(filters) like : 

Import only,
1)Read emails,
2)Unread emails,
3)Received before date emails,
4)Received after date emails,
5)Received on date emails
6)Sent emails,etc.  before importing all emails.

It will save our time as well as memory.

Thanks.

**Comment by pgorod** [comment]
This might sound like an enhancement request but I don't know if it shouldn't be called a bug. At least the simple option not to import anything is "a must". I would hate to thousands of emails brought into the database unwillingly.

**Comment by psatasiya** [comment]
Yes,It should not be called a bug,but it could be a future enhancement.

**Comment by pgorod** [comment]
@psatasiya we're not agreeing, on that we're saying opposite things. : - )

I have 7400 emails in my Gmail Inbox. I would NOT want them in the SuiteCRM database. Basically, that means I cannot connect my Gmail with SuiteCRM!

Multiply this by dozens or hundreds of users, and you might have a deal-breaker in this issue. Maybe it is not strictly a bug (code that doesn't work as expected), but it is something that needs to be fixed, a mandatory enhancement. IMHO.

**Comment by samus-aran** [comment]
@pgorod A note here that on the Inbound settings when you create a new Mail account you can unselect 'Import Emails Automatically'  Also you can select the folder you wish to import i.e not your inbox.

But yes, this would be a suggestion to further enhance importing emails.

**Comment by samus-aran** [comment]
Actually I follow up here.  If you do not select the 'Import Emails Automatically' you can still view your emails and manually Import them by Right click the email and seeing 'Import to SuiteCRM'

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Configuration of new inbound email account imports all emails](https://trello.com/c/TeG6f1gN/88-configuration-of-new-inbound-email-account-imports-all-emails)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
